src/DurableTask.SqlServer/SqlOrchestrationService.cs

```diff
@@ -19,6 +19,7 @@ namespace DurableTask.SqlServer
     using DurableTask.Core.Query;
     using DurableTask.SqlServer.SqlTypes;
     using DurableTask.SqlServer.Utils;
+    using Dynamitey.DynamicObjects;
```

Comment:
Is this being used or can it be removed?

Reply:
Fixed